src/utils.js

```diff
@@ -50,6 +50,20 @@ const isMaskJsonVendor = vendor => {
 const isVendorMaskValid = maskVendor =>
   isIronMaskVendor(maskVendor) || isMaskJsonVendor(maskVendor)
 
+const filterLargeProp = (object, prop, propLengthLimit) => {
+  if (!object[prop]) return
+
+  return JSON.stringify(object[prop]).length > propLengthLimit ? {} : object[prop]
```

Comment:
E se `object[prop]` for um array mto grande, não deveríamos retornar `[]` ?

Reply:
Não sei se entendi, `object[prop]`  retorna um objeto, eu converto para uma string e vejo se o length excede ao limite, o único array que  existe é o propsToLog mas isso fica no `src/[request/response]-logger.js`

Reply:
Se você diz de colocar um limite para  o tamanho da string, o que você sugere :upside_down_face: ?

Reply:
Nesse ternário aqui `JSON.stringify(object[prop]).length > propLengthLimit ? {} : object[prop]`, vc retorna sempre um objeto vazio (`{}`) se passar o tamanho limite. Se for um array, não deveria ser `[]` caso passe o tamanho ?